index.html

```diff
@@ -53,6 +53,22 @@ <h6>
 				</p>
 			</div>
 		</div>
+		<div class="card col-md-4 col-xs-12" style="width: 18rem;">
+            <img src="Assets/ayushisrivastava.jpeg" class="card-img-top" alt="Ayushi-Srivastava">
+            <div class="card-body">
+                <h5 class="card-title">
+                    Ayushi Srivastava <a class="fa fa-linkedin" href="https://www.linkedin.com/in/srivayushi/" target="_blank"></a> <a class="fa fa-twitter" href="https://twitter.com/srivayushi12" target="_blank"></a> <a class="fa fa-instagram" href="https://www.instagram.com/_ayyuuuu_/" target="_blank"></a> <a class="fa fa-link" href="https://www.ayushi.live/" target="_blank"></a> 
+                </h5>
+                <h6>
+                    GDSC Core Team - Amity University, Noida
+                </h6>
+                <p class="card-text">
+                    GDSC Amity has provided me the support and platform to showcase my skills ! Happy to be a part of GDSC.
+                </p>
+            </div>
+          </div>
```

Comment:
This is an extra `</div` that is messing with the page styling. Please remove it and test the changes before pushing.

Preview from your commit: https://srivastavayushi-members-showcase-j99g76qr25rpr-5500.githubpreview.dev/